libr/crypto/crypto.c

```diff
@@ -72,9 +72,11 @@ R_API bool r_crypto_use(RCrypto *cry, const char *algo) {
 	r_list_foreach (cry->plugins, iter, h) {
 		if (h && h->use && h->use (algo)) {
 			cry->h = h;
-			cry->key_len = h->get_key_size (cry);
-			cry->key = calloc (1, cry->key_len);
-			return cry->key != NULL;
+			if (h->get_key_size) {	//should i change this or make base64/base91 return 0 for get_key_size?
```

Comment:
Need feedback whether I should add `get_key_size` function for algorithms which don't require key for encoding/decoding and return 0 value or this change is correct.